src/osmf/lib/HDSPlayer.as

```diff
@@ -834,7 +839,15 @@ package
     public function onGetCurrentTime(event:Event):void
     {
       var eventObject:Object = new Object();
-      eventObject.currentTime = _mediaPlayerSprite.mediaPlayer.currentTime.toString();
+      if (_seekToEnd == true)
+      {
+        eventObject.currentTime = _mediaPlayerSprite.mediaPlayer.duration.toString();
+        _seekToEnd = false;
```

Comment:
If I could call getCurrentTime at any time, what does getCurrentTime have to do with seekToEnd?  Why does it get reset to false here?  Shouldn't it remain false until we are actually not at the end anymore? 

Also this variable is now specific to seeking but what if I call getCurrentTime when you didn't seek?  The behaviour of getCurrentTime shouldn't depend on how you go to the end of the stream; just if you got there